### DISCUSSION TITLE
Build OVMS firmware for the wican/wican pro hardware

### DISCUSSION THREAD

**Discussion by fowi4hjte** [meatpiHQ/wican-fw#209] [General]
For longer time it was not possible to build a firmware for the meatPI/wican (pro) hardware because OVMS was stucked at ESP-IDF version 3.3. But since Aug 3, 2024 things changed and hopefully a build would be possible now.
The WICAN Pro is really the perfect upgrade replacement for the OVMS hardware. It have the perfect amount of ram/rom and the exact chipset (esp32-s3) the ovms people was thinking of using to create a new OVMS hardware revision.
Also the existing meatpi/wican hardware would already make it possible to run OVMS firmware. The upcomming pro can fully replace the OVMS hardware because it supports optional LTE and gps.

I would be interested in ordering a wican device to run it with OVMS firmware.

**Comment by meatpiHQ** [comment]
@fowi4hjte I've answered [here ](https://github.com/meatpiHQ/wican-fw/discussions/56)

I don't mind doing it, if OVMS are ok with it.
